README.md

```diff
@@ -90,19 +90,118 @@ The load operation may take up to a few of seconds. Load Ver-ID using the `load`
+          throw 'Session canceled';
+        }
+        return value;
+      });
+    } else {
+      throw verIdNotInitialized;
+    }
+  }
+  
+  Future<SessionResult> registerUser(String userId) async {
+    if (verID != null) {
+      RegistrationSessionSettings settings =
+          new RegistrationSessionSettings(userId: userId);
+      settings.showResult = true;
+      return VerID.register(settings: settings).then((value) {
+        if (value == null) {
+          throw 'Session canceled';
+        }
+        return value;
+      });
+    } else {
+      throw verIdNotInitialized;
+    }
+  }
+~~~
+
+
+### Liveness Detection
+
+In a liveness detection session the user is asked to assume a series of random poses in front of the camera.
 
-### Face detection session without asking for poses
+Liveness detection sessions follow the same format as registration and authentication.
 
-### Liveness detection session defining the bearings (poses) the user may be asked to assume
+## Extracting faces for face comparison
+
+~~~dart
+Future<SessionResult> captureLiveFace() async {
+    if (verID != null) {
+      LivenessDetectionSessionSettings settings =
+          new LivenessDetectionSessionSettings();
+      return VerID.captureLiveFace(settings: settings).then((value) {
+        if (value == null) {
+          throw 'Session canceled';
+        }
+        return value;
+      });
+    } else {
+      throw verIdNotInitialized;
+    }
+  }
+~~~
+
+## Face detection session without asking for poses
+
+~~~dart
+Future<SessionResult> captureLiveFace() async {
+    if (verID != null) {
+      LivenessDetectionSessionSettings settings =
+          new LivenessDetectionSessionSettings();
+      return VerID.captureLiveFace(settings: settings).then((value) {
+        if (value == null) {
+          throw 'Session canceled';
+        }
+        return value;
+      });
+    } else {
+      throw verIdNotInitialized;
+    }
+  }
+~~~
 
 ## Session Response Format
+The callback of a successful session will contain [an object](https://appliedrecognition.github.io/Ver-ID-Person-Cordova-Plugin/classes/_ver_id_.sessionresult.html) that represents the result of the session.
 
 ## Comparing Faces
 
+~~~dart
+Future<FaceComparisonResult> compareFaces(Face face1, Face face2) async {
+    if (verID != null) {
+      return VerID.compareFaces(face1: face1, face2: face2);
+    } else {
+      throw verIdNotInitialized;
+    }
+  }
+~~~
+
 ## Detecting Faces In Images
 
+~~~dart
+Future<Face> detectFaceInImage(String imageData) async {
+    if (verID != null) {
+      return VerID.detectFaceInImage(image: imageData);
+    } else {
+      throw verIdNotInitialized;
+    }
+  }
+~~~
+
 ## Project Samples
+Project samples can be found on the samples branch of this repository, to be used as a reference for development purposes.
 
 ## Module API Reference
-
+- [Ver-ID](https://appliedrecognition.github.io/Ver-ID-Person-Cordova-Plugin/modules/_ver_id_.html)
```

Comment:
Cordova link

Reply:
@jakubdolejs we'll need to move this to be serviced from https://github.com/AppliedRecognition/Ver-ID-Flutter/blob/develop/doc/api/index.html later on, or creating a Github page to load directly.